<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_widget</class>
 <widget class="QWidget" name="login_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travelfimov | Login</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="login_widget_layout">
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Имя пользователя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="placeholderText">
       <string>login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="text">
       <string>Вход</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="loginMessage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
